aggregators/1.0/modules/ROOT/pages/aggregators-module-reference.adoc

```diff
@@ -60,49 +59,48 @@ There are size-based, time-based, and group-based aggregators.
 | Content | Expression | The expression that defines what to aggregate. The result of the evaluation is the value stored in the aggregation. | #[payload] |
-| Max Size | Number | The total number of elements to be aggregated before considering the aggregation complete. | | x
-| Timeout | Number |  A maximum time to wait for the aggregation to complete. If the timeout is reached before the total number of elements is equal to max size, the aggregation is not considered complete. A value of `0` is not supported to avoid timing out the group constantly. | -1(UNLIMITED) |
+| Max Size | Number | The total number of elements to aggregate before considering the aggregation complete. | | x
+| Timeout | Number |  A maximum time to wait for the aggregation to complete. If the timeout is reached before the total number of elements is equal to Max Size, the aggregation is not considered complete. A value of `0` is not supported, to avoid constant group timeouts. | -1(UNLIMITED) |
 | Timeout unit | Time Unit | The time unit in which to measure the timeout. |  SECONDS |
 | Object Store | Object Store |  Either a name to reference a global object store or a definition of a private object store where the aggregated elements will be stored. |  Default Object Store partition. |
 | Aggregation Complete Route | Route | Components chain to execute once the aggregation is complete. The Aggregation Complete Route does not execute if the timeout is reached. | | x
-| Incremental Aggregation Route | Route | Components chain to execute for every new element that is aggregated. The payload is the list of all the elements that have been aggregated from the first one (in this aggregation) to the one that is being aggregated now. | |
+| Incremental Aggregation Route | Route | Components chain to execute for every new element that is aggregated. The payload is the list of all the elements that have been aggregated in the aggregation from the first element to the one that is currently aggregated. | |
 |===
 
 
 ==== Raises
 
 * AGGREGATORS:AGGREGATOR_CONFIG
-+
-Whenever maxSize or timeout have invalid values (for example, `maxSize < 0`).
-+
+
+The Max size or Timeout fields have invalid values, for example, `maxSize < 0`.
+
 * AGGREGATORS:OBJECT_STORE_ACCESS
-+
-If there is any issue accessing the ObjectStore used to store the aggregated values.
 
----
-image::size-based-aggregator.png[Size based Aggregator]
-. Input Data
-. Aggregated elements (Not complete)
-. Aggregated elements (Complete)
-. Input Data (same as 1) + New Variables
-. Aggregated elements, one of :
-.. `maxSize` reached (same as 3)
-.. `timeout` (only if timed out groups are accepted)
+An error occurred during an attempt to access the object store used to store the aggregated values.
 
----
 
 [[time-based-aggregator]]
 === Time-Based Aggregator
 
+The *Time based aggregator* scope enables you to aggregate elements until a specified time limit is reached, executing the routes and listeners.
```

Comment:
... until a specified time limit is reached when executing the routes and listeners?